lib/mapRoutes.js

```diff
@@ -60,16 +62,40 @@ module.exports = function (app) {
 
   // make public folder itself if it doesn't exist
   if (!fileExists(publicDir)) {
-    fse.mkdirsSync(publicDir)
-    logger.log('📁', `${appName} making new directory ${publicDir}`.yellow)
+    if (fsr.ensureDirSync(publicDir)) {
+      logger.log('📁', `${appName} making new directory ${publicDir}`.yellow)
+    }
   }
 
   // make statics prefix folder if the setting is enabled
   if (params.staticsPrefix) {
     publicDir = path.join(publicDir, params.staticsPrefix)
     if (!fileExists(publicDir)) {
-      fse.mkdirsSync(publicDir)
-      logger.log('📁', `${appName} making new directory ${publicDir}`.yellow)
+      if (fsr.ensureDirSync(publicDir)) {
+        logger.log('📁', `${appName} making new directory ${publicDir}`.yellow)
+      }
+    }
+  }
+
+  // if js and/or css compilers are disabled, check that jsPath and cssPath are set to be symlinked
+  if (params.cssCompiler === 'none' || params.cssCompiler === null) {
+    basicCSSPath = params.cssPath.replace(params.staticsRoot, '')
+    if (basicCSSPath.charAt(0) === path.sep) {
+      basicCSSPath = basicCSSPath.substr(1)
+    }
+    if (!(basicCSSPath in params.symlinksToStatics)) {
+      logger.warn('Adding CSS directory to symlinksToStatics list'.yellow)
+      params.symlinksToStatics.unshift(basicCSSPath)
+    }
+  }
+  if (params.jsCompiler === 'none' || params.jsCompiler === null) {
+    basicJSPath = params.jsPath.replace(params.staticsRoot, '')
+    if (basicJSPath.charAt(0) === path.sep) {
+      basicJSPath = basicJSPath.substr(1)
+    }
+    if (!(basicJSPath in params.symlinksToStatics)) {
+      logger.warn('Adding JS directory to symlinksToStatics list'.yellow)
```

Comment:
Make this a log, not a warning. And make it say (in blue symlink log text): "📁  [appname] making new symlink [public path in yellow] pointing to [statics path in yellow]"